files/callback/lock_run.py

```diff
@@ -37,7 +37,7 @@ class CallbackModule(CallbackBase):
     def __init__(self):
         super(CallbackModule, self).__init__()
 
-    def v2_playbook_on_play_start(self, play):
+    def v2_playbook_on_start(self, play):
```

Comment:
The variable is called **playbook**. I know it is unused at the moment, but it is disturbing if you just discovered the plugins API like me now.